labgrid/util/ssh.py

```diff
@@ -472,9 +476,7 @@ def _stop_keepalive(self):
             self._keepalive.communicate(timeout=60)
         except subprocess.TimeoutExpired:
             self._keepalive.kill()
-
-        try:
-            self._keepalive.wait(timeout=60)
+            self._keepalive.communicate()
```

Comment:
Is dropping the timeout here intentional?

Reply:
Yes, I expect the process that received SIGKILL to terminate soon, so we don't need a timeout.

Reply:
Still, it should not wait forever if something goes wrong.

Reply:
Okay, re-added `timeout=60`.